lmdeploy/pytorch/check_env/__init__.py

```diff
@@ -87,6 +87,27 @@ def check_env(device_type: str):
 MAX_TRANSFORMERS_VERSION = '4.41.2'
 
 
+def check_awq(hf_config):
+    """check awq support."""
+    logger = get_logger('lmdeploy')
+    quantization_config = getattr(hf_config, 'quantization_config', dict())
+    quant_method = quantization_config.get('quant_method', None)
+    if quant_method != 'awq':
+        return
+    try:
+        import awq  # noqa
+    except Exception as e:
+        _handle_exception(e, 'awq', logger)
+
+    try:
+        import awq_ext  # noqa
```

Comment:
what's the difference between awq and awq_ext?

Reply:
https://github.com/casper-hansen/AutoAWQ/tree/main
https://github.com/casper-hansen/AutoAWQ_kernels